### PR TITLE
[plugin.video.putio] 3.0.1

### PR DESCRIPTION
**Pull request by berkanteber**
### Description

This update adds compatibility with Kodi v20.

#### Changes ([diff](https://github.com/putdotio/putio-kodi/compare/c3c0b967..master)):
- `xbmcvfs.translatePath` is used instead of `xbmc.translatePath`.

### Checklist:

- [X] My code follows the [add-on rules](http://kodi.wiki/view/Add-on_rules) and [piracy stance](http://kodi.wiki/view/Official:Forum_rules#Piracy_Policy) of this project. 
- [X] I have read the [CONTRIBUTING](https://github.com/xbmc/repo-plugins/blob/master/CONTRIBUTING.md) document
- [X] Each add-on submission should be a single commit with using the following style: [plugin.video.foo] v1.0.0

Additional information :
- Submitting your add-on to this specific branch makes it available to any Kodi version equal or higher than the branch name with the applicable Kodi dependencies limits.
- [add-on development](http://kodi.wiki/view/Add-on_development) wiki page.
- Kodi [pydocs](http://kodi.wiki/view/PyDocs) provide information about the Python API
- [PEP8](https://www.python.org/dev/peps/pep-0008/) codingstyle which is considered best practice but not mandatory.
- This add-on repository has automated code guideline check which could help you improve your coding. You can find the results of these check at [Codacy](https://www.codacy.com/app/Kodi/repo-plugins/dashboard). You can create your own account as well to continuously monitor your python coding before submitting to repo.
- Development questions can be asked in the [add-on development](http://forum.kodi.tv/forumdisplay.php?fid=26) section on the Kodi forum.
- If you see no activity on your PR after a week (so at least one weekend has passed) then please go to the #kodi-dev freenode IRC channel to reach out to the team

### CODE REVIEW

**Review by BatuAksoy — APPROVED** [review]

**Comment by basrieter** [comment]
Would it not make sense to PR this to the Matrix branch? Then the changes will automatically be available in Matrix and Nexus?

**Comment by berkanteber** [comment]
It is deprecated but not removed in Matrix, so it still works there. I thought it would be unnecessary to release a new version when something is not broken.

I could still add it to Matrix as you said if you think it is better. Up to you.

**Comment by basrieter** [comment]
Do they differ per Kodi version?

**Comment by berkanteber** [comment]
Sorry, I couldn't understand the question (do _what_ differ?). 

Old addon uses `xbmc.translatePath()`, new addon uses `xbmcvfs.translatePath()`.

- Pre-Matrix: Only `xbmc.translatePath()` is OK.
- Matrix: Both are OK. Former is deprecated in favor of latter, but not removed yet.
- Nexus: Only `xbmcvfs.translatePath()` is OK.

We could release this version (fully identical) on both Matrix & Nexus. I opted for Nexus only. But releasing for Matrix is also completely fine.

**Comment by basrieter** [comment]
I would go for a single release. That would make reviewing easier. We really want to get rid of duplicate add-ons in Kodi version if it is not needed.

**Comment by berkanteber** [comment]
I meant a single release, either one is fine (1 or 2, not both).

1. Release for Nexus as in this PR.
2. Release for Matrix (automatically available to Nexus).

**Comment by berkanteber** [comment]
Now that Nexus is officially released, we've started getting complaints from our users about this issue. Can we please speed up this PR? We are fine either releasing to Nexus (this PR as is), or release to Matrix (I can update the PR to merge to that branch).

**Comment by muraty** [comment]
Hello @basrieter. Can you please check it? Thanks.

File: plugin.video.putio/addon.xml [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+    <requires>
+        <import addon="xbmc.python" version="3.0.0" />
+        <import addon="script.module.requests" version="2.12.4" />
+    </requires>
+    <extension point="xbmc.python.pluginsource" library="main.py">
+        <provides>video</provides>
+    </extension>
+    <extension point="xbmc.service" library="service.py" />
+    <extension point="xbmc.addon.metadata">
+        <summary lang="en">Put.io for Kodi</summary>
```
Comment (enen92) [inline plugin.video.putio/addon.xml]: en_GB

File: plugin.video.putio/addon.xml [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+        <description lang="en">Put.io is a service that combines cloud storage and bittorrent. You need an account to use this plug-in. Get one from https://put.io</description>
```
Comment (enen92) [inline plugin.video.putio/addon.xml]: en_GB

File: plugin.video.putio/addon.xml [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+        <disclaimer lang="en_GB">Please read terms of service at https://put.io/terms-of-service/</disclaimer>
```
Comment (enen92) [inline plugin.video.putio/addon.xml:18]: en_GB

File: plugin.video.putio/resources/__init__.py [hunk 4]
```diff
@@ -0,0 +1,16 @@
+# put.io kodi addon
```
Comment (enen92) [inline plugin.video.putio/resources/__init__.py]: should not be needed anymore with python3 please drop it

**Review by enen92 — CHANGES_REQUESTED** [review]

File: plugin.video.putio/addon.xml (hunk 1 above)
Comment (berkanteber) [reply plugin.video.putio/addon.xml]: 298878a9

File: plugin.video.putio/addon.xml (hunk 2 above)
Comment (berkanteber) [reply plugin.video.putio/addon.xml]: 298878a9

File: plugin.video.putio/addon.xml (hunk 3 above)
Comment (berkanteber) [reply plugin.video.putio/addon.xml:18]: This was already `en_GB`.

File: plugin.video.putio/resources/__init__.py (hunk 4 above)
Comment (berkanteber) [reply plugin.video.putio/resources/__init__.py]: f1a18eac

**Review by enen92 — APPROVED** [review]

**Pull request merged**